### DISCUSSION TITLE
What's the size of the Visual Studio Extension?

### DISCUSSION THREAD

**Discussion by benoriol** [James-Yu/LaTeX-Workshop#3522] [Q&A] [answered]
There are some compilers like MacTex that take like 5Gb of disk memory. How much memory does LaTeX-Workshop take? Or maybe it also uses mactex under the roof?

Thanks,
Benet

**Comment by jlelong** [comment, marked as answer]
You need to install a LaTeX distribution https://github.com/James-Yu/LaTeX-Workshop/wiki/Install#requirements
LaTeX-Workshop just turns VS Code into a LaTeX IDE.

**Comment by benoriol** [comment]
Thanks, I see it does in fact use MacTex under the hood.
